distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> My preference would go to extending the dashboard a little bit with a default admin/feature installation page

I was thinking about that once as well, but then decided to include the "extension management in UI" only in the Paper UI, which will be present for a "normal" setup and it comes with all nice UI features like event, touch device support etc.

Two further reasons why purely relying on an admin UI is not good:
- It means that the minimal runtime already requires an http server, which is actually pretty big. If you think of an "headless" openHAB instance, e.g. for only running a specific binding and sending data through MQTT, an http server is not needed.
- There must be a way to specify your setup, so that it can be backed up or automatically constructed. This is why I introduced the "addons.cfg" file already.

> This would also allow us to not specify all repository urls by default, but add them programmatically when needed

This again is rather a configuration setting for me (use snapshots or not) than a UI functionality. Of course you can also provide such features in the UI later on, but the basis must be a configuration setting.

> they would never have to download a new runtime again

This is something that I would leave out of scope for the moment - it would already be a big gain if we can easily update the bindings/extensions. Updating the runtime itself will be much more difficult as you would also have to cover files like "start.sh" or whatsoever. For the moment the upgrade should be easy enough by simply allowing to manually replace the "runtime" folder, while keeping the "conf" and "userdata" folder in place.